src/h5/dd-resizable.ts

```diff
@@ -152,6 +155,8 @@ export class DDResizable extends DDBaseImplement implements HTMLElementExtendOpt
   /** @internal */
   private _resizeStart(event: MouseEvent): DDResizable {
     this.originalRect = this.el.getBoundingClientRect();
+    const scrollEl = Utils.getScrollParent(this.el);
```

Comment:
cache this since you need it in all 3 methods and continuously, or does it change over time ?

Reply:
This changes over time. When there is no scroll because the layout is small `scrollEl` is null and eventually when the layout grows returns the scrollable element.

Reply:
right, so you could `scrollEl = scrollEl || Utils.getScrollParent(this.el)` and cache it once it appears, unless it changes values ?
quick looking it appears to go up recursively so maybe it could be different elements over time...